src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -458,6 +458,9 @@ public MessageBuilder appendCustomMessage(Result buildResult) {
                     customMessage = notifier.getCustomMessageFailure();
                 }
             }
+            if (customMessage.isEmpty()) {
```

Comment:
can you also look at removing the check box from custom message so it is consistent with the other new fields?
thanks

Reply:
Which checkbox do you mean? the `includeCustomMessage` checkbox? This is applied to all custom messages, it works as expected afaik. Do you mean something else?

Reply:
you're right never mind